apiv2/schema/schema.yaml

```diff
@@ -1771,6 +1780,111 @@ classes:
         unit:
           symbol: °
           descriptive_name: degrees
+  PerSectionParameters:
+    name: PerSectionParameters
+    annotations:
+      plural:
+        tag: plural
+        value: PerSectionParameters
+    description: Map individual Frames to a Tiltseries
+    from_schema: cdp-dataset-config
+    mixins:
+      - IDMixin
+    attributes:
+      astigmatic_angle:
+        name: astigmatic_angle
+        description: Angle (in degrees) from reciprocal space X axis to the major axis of defocus.
+        from_schema: cdp-dataset-config
```

Comment:
I think we don't need the `from_schema` here and the other entries below because this is not referencing a definition from else where, but rather defining the attribute here.